styles.css

```diff
@@ -0,0 +1,1014 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+.line-bar {
+  position: absolute;
+  height: 1px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin-right: 10px;
+  margin-left: 20px;
+  padding-left: 50px;
+  padding-right: 30px;
+  background: #6f6c6b;
+}
+
+.accordion {
+  background-color: #eee;
+  color: #444;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
+  border: none;
+  text-align: left;
+  outline: none;
+  font-size: 15px;
+  transition: 0.4s;
+}
+
+.close-but {
+  margin-left: 275px;
+}
+
+.mobile-nav {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  right: 0;
+  overflow-x: hidden;
+  transition: 0.5s;
+  padding-top: 60px;
+  background-color: #3c3a39;
+}
+
+.mobile-nav a {
+  padding: 8px 8px 8px 32px;
+  text-decoration: none;
+  font-size: 32px;
+  transition: 0.3s;
+  margin-left: 28px;
+  color: #fff5e1;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  height: 50px;
+}
+
+.mobile-nav a:hover {
+  color: #f1f1f1;
+}
+
+.display-popup,
+.display-popup-container {
+  display: none;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: fixed;
+  width: 88%;
+  top: 12px;
+}
+
+.display-popup {
+  top: 0;
+  height: 100vh;
+  left: 0;
+  width: 100vw;
+  background: rgba(151, 148, 147, 0.3);
+  backdrop-filter: blur(10px);
+}
+
+.popup-nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup-img {
+  width: 100%;
+}
+
+.display-popup-container {
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  display: block;
+  height: 100vh;
+  overflow: scroll;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+  padding: 0 50px;
+}
+
+.header {
+  background-color: #3c3a39;
+  height: 9vh;
+}
+
+.header-bar {
+  color: #fff;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+.header .header-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
- Following the project requirements, it is important to adhere as closely as possible to the Figma design. However, I have noticed that the color of your hamburger menu does not match the specified color in the design. Could you kindly address this matter and ensure that the color of the hamburger menu is consistent with the Figma design? 🎯
### Your Design:
![image](https://user-images.githubusercontent.com/107952724/233201874-54f05b7c-db0f-4929-bf4e-05b511aaa0b9.png)
### Figma Template:
![image](https://user-images.githubusercontent.com/107952724/233202202-ec2ec5fa-ef35-4194-9e64-beaf02eee338.png)